src/lambda/commands/getLambdaPolicy.ts

```diff
@@ -14,29 +18,50 @@ import { LambdaTemplates } from '../templates/lambdaTemplates'
 import { getSelectedLambdaNode } from '../utils'
 
 export async function getLambdaPolicy(awsContext: AwsContext, element?: FunctionNode) {
+    let functionName: string = 'Unknown'
+    let view: vscode.WebviewPanel | undefined
+    const baseTemplateFn = _.template(BaseTemplates.SIMPLE_HTML)
+
     try {
         const fn: FunctionNode = await getSelectedLambdaNode(awsContext, element)
+        functionName = fn.functionConfiguration.FunctionName || functionName
```

Comment:
Note that an empty string is falsy in JS, so this will default to `functionName` is `functionConfiguration.FunctionName` is empty. Is that intentional?